MaxText/pyconfig.py

```diff
@@ -149,7 +149,9 @@ def validate_model_name(s: str) -> bool:
       "mixtral-8x7b",
       "gemma-7b",
       "gemma-2b",
+      "gemma2-2b",
       "gemma2-9b",
+      "gemma2-27b",
```

Comment:
nit: Should we remove this key for now until we add the gemma2-27b config yml file ? Just don't want people to assume gemma-27b is supported now.

Reply:
added gemma2-27b config yaml file, thanks for the review!

Reply:
Have we tested golden logits for the gemma2-27b as well?

Reply:
added the 2b and 9b, for 9b I tested it on `JAX_PLATFORMS=cpu` with `dtype=float32` locally and it works, for 27b it OOMs even on CPU with scanned checkpoint so removing it from the `pyconfig` file for now